plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java

```diff
@@ -534,7 +534,7 @@ private void setPreviousSelection() {
         targetPath = targetPath.substring( 7 );
       }
       java.io.File filePath = new java.io.File( targetPath );
```

Comment:
I don't agree with the solution as it doesn't address the core of the problem using` java.io.File` which is for local files URI(`"C:\someDirectory"` or `"/someDirectory"` ) for PVFS URI's ie (`"pvfs://someConneciton/someFile"`)

**It may address** the original case by changing (ProviderFilterType.REPOSITORY to  ProviderFilterType.LOCAL), but a new developer should question why am I using java.io.File for PVFS URI. Then you'd have to either add a java comment and/or they would have to surmise all the edge cases and determine if the code is still valid.

If ever there are changes to this function in the future, the core problem will still be there. hopefully they don't need  to use PVFS URI ~ exists() or PVFS URI ~ getParent() function

if these changes are a means to unblock efforts for "project-profile" by all means go ahead. But I would not consider this issue addressed

running a simple unit test below, the first two asserts pass, and the last two fail

`java.io.File#exists` doesn't behave properly for PVFS URI
`java.io.File#getParent` doesn't behave properly for the PVFS URI

```
@Test
public void testJavaIOFIle() throws Exception {

  Assert.assertFalse( new java.io.File(  "pvfs://someConnection/folder-that-does-not-exist").exists() );
  Assert.assertFalse( new java.io.File(  "pvfs://someConnection/folder-that-does-not-exist").isFile() );
  Assert.assertEquals("pvfs://someConnection", new java.io.File(  "pvfs://someConnection/folder-that-does-not-exist").getParent() );
  Assert.assertTrue( new java.io.File(  "pvfs://someConnection/valid-folder").exists() );

}
```

Reply:
I've pushed a newer update that makes it so we never use java.io.File to parse any non-LOCAL URIs.